neuralprophet/forecaster.py

```diff
@@ -1824,22 +1844,27 @@ def plot_latest_forecast(
             fcst, self.config_train.quantiles, n_last=1 + include_previous_forecasts
         )
 
-        # Check whether the default plotting backend is overwritten
+        # Check whether a local or global plotting backend is set.
         plotting_backend = (
-            plotting_backend
-            if plotting_backend != "default"
-            else (self.plotting_backend if hasattr(self, "plotting_backend") else "matplotlib")
+            auto_set_plotting_backend(plotting_backend)
+            if plotting_backend is not None
+            else (
+                auto_set_plotting_backend(self.plotting_backend)
+                if hasattr(self, "plotting_backend")
+                else auto_set_plotting_backend("plotly-auto")
+            )
         )
         log_warning_deprecation_plotly(plotting_backend)
-        if plotting_backend == "plotly":
+        if plotting_backend.find("plotly") == 0:
```

Comment:
Suggestion: use more pythonic notation:
`if 'plotly' in plotting_backend.lower():`